engine/org.eclipse.birt.report.engine.emitter.wpml/src/org/eclipse/birt/report/engine/emitter/wpml/AbstractEmitterImpl.java

```diff
@@ -1055,6 +1057,28 @@ public void startImage(IImageContent image) {
 				float scale = ((float) imageInfo.getHeight()) / ((float) imageInfo.getWidth());
 				height = width * scale;
 			}
+System.out.println("==== standard ==== ");
```

Comment:
Debugging code?

Reply:
Confirmed, I will remove the output lines.